### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by sachsensoft** [community/community#72811] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have updated image on repo quite long time ago but github still remember the old version. 

![image](https://github.com/community/community/assets/127749241/ce92700f-49d4-471f-8feb-066a99378527)

How to force to refresh it?

**Comment by ktauchathuranga** [comment]
do you mean in commit history?

**Reply by sachsensoft** [reply]
No, just in repo, in Gitlab Web GUI, when you click on file, gitlab shows old (top white one on screenshot), but when you download it, it will be current real one (black one).
